src/spas/libs/member-management/feature/src/lib/member-management/user-detail/user-detail.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { UserProfile } from '../../../../../domain/src/lib/models/user-profile';
```

Comment:
When consuming types from other internal libs, use the @nimbes import: [example](https://github.com/nimb3s/nimb3s/blob/b196c7ef963cd95a9a6c1fa4eb5ec7d2d9240433/src/spas/libs/shared/ui/layout/src/lib/toolbar/toolbar.module.ts#L6)